internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -262,6 +267,10 @@ func SchemaAdvancedConfig() *schema.Schema {
 					Optional: true,
 					Default:  -1,
 				},
+				"default_max_time_ms": {
+					Type:     schema.TypeInt,
+					Optional: true,
```

Comment:
[Confirmed with upstream](https://jira.mongodb.org/browse/CLOUDP-250898?focusedCommentId=6890815&focusedId=6890815&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6890815) this is expected to be user-managed